lib/cdo/aws/cloudfront.rb

```diff
@@ -0,0 +1,211 @@
+require_relative '../../../deployment'
```

Comment:
Added CloudFront unit tests to [test_cloudfront.rb](https://github.com/code-dot-org/code-dot-org/blob/b781b19ee8812d56eaeaeb0306d784772571fdb1/shared/test/test_cloudfront.rb), with the following note:

> These unit tests simply confirm that the #create_or_update method will pass
properly-structured data to the AWS CloudFront Client library.
Separate integration tests are required to guarantee that the live API
endpoints will accept the values provided.

I think manual testing of the CloudFront distribution created by this script should be fine for now, and we can look into adding integration tests later if possible.